authors/apps/articles/serializers.py

```diff
@@ -101,6 +101,7 @@ class CommentSerializer(serializers.ModelSerializer):
     """Serializer for creating a Comment"""
     author = serializers.SerializerMethodField()
     body = serializers.CharField()
+    selected_text = serializers.CharField(read_only=True)
```

Comment:
I think a different serializer will be created for this purpose... per my comment above.

Reply:
Okay noted